internal/pkg/agent/application/upgrade/step_unpack.go

```diff
@@ -34,15 +35,15 @@ type UnpackResult struct {
 }
 
 // unpack unpacks archive correctly, skips root (symlink, config...) unpacks data/*
-func (u *Upgrader) unpack(version, archivePath, dataDir string) (UnpackResult, error) {
+func (u *Upgrader) unpack(version, archivePath, dataDir string, skipComponents bool, flavor string) (UnpackResult, error) {
```

Comment:
What is `skipComponents` for? It doesn't look like it's used in the function body.

Weird that none of our linters catches this.

Reply:
i thought i removed it, thanks